int8 NOMODE=0
int8 DISABLED=1
int8 ESTOP=2
int8 TELEOP=3
int8 AUTONOMOUS=4
int8 TEST=5
int8 mode
---
bool success
string message
